### PR TITLE
Add switch to github and electron releases

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [x] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add the ability to release desktop workflow to enable/disable release electron to R2 and creation of github release.

## Code changes

- **.github/workflows/release-desktop.yml:** Added two checkbox inputs to enable/disable electron and github deployment

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: .github/workflows/release-desktop.yml [hunk 1]
```diff
@@ -136,6 +146,7 @@ jobs:
         run: mv Bitwarden-${{ env.PKG_VERSION }}-universal.pkg Bitwarden-${{ env.PKG_VERSION }}-universal.pkg.archive
 
       - name: Set staged rollout percentage
+        if: ${{ inputs.electron_publish }}
```
Comment (joseph-flinn) [inline .github/workflows/release-desktop.yml:149]:
```suggestion
        if: ${{ github.event.inputs.electron_publish }}
```

Looks like the rest of the inputs use `github.event.inputs.*`

File: .github/workflows/release-desktop.yml [hunk 2]
```diff
@@ -145,7 +156,7 @@ jobs:
           echo "stagingPercentage: ${ROLLOUT_PCT}" >> apps/desktop/artifacts/${RELEASE_CHANNEL}-mac.yml
 
       - name: Publish artifacts to S3
-        if: ${{ github.event.inputs.release_type != 'Dry Run' }}
+        if: ${{ github.event.inputs.release_type != 'Dry Run' && inputs.electron_publish }}
```
Comment (joseph-flinn) [inline .github/workflows/release-desktop.yml:159]:
```suggestion
        if: ${{ github.event.inputs.release_type != 'Dry Run' && github.event.inputs.electron_publish }}
```

File: .github/workflows/release-desktop.yml [hunk 3]
```diff
@@ -159,7 +170,7 @@ jobs:
           --quiet
 
       - name: Publish artifacts to R2
-        if: ${{ github.event.inputs.release_type != 'Dry Run' }}
+        if: ${{ github.event.inputs.release_type != 'Dry Run' && inputs.electron_publish }}
```
Comment (joseph-flinn) [inline .github/workflows/release-desktop.yml:173]:
```suggestion
        if: ${{ github.event.inputs.release_type != 'Dry Run' && github.event.inputs.electron_publish }}
```

File: .github/workflows/release-desktop.yml [hunk 4]
```diff
@@ -175,7 +186,7 @@ jobs:
 
       - name: Create Release
         uses: ncipollo/release-action@95215a3cb6e6a1908b3c44e00b4fdb15548b1e09
-        if: ${{ steps.release-channel.outputs.channel == 'latest' && github.event.inputs.release_type != 'Dry Run' }}
+        if: ${{ steps.release-channel.outputs.channel == 'latest' && github.event.inputs.release_type != 'Dry Run' && inputs.github_release }}
```
Comment (joseph-flinn) [inline .github/workflows/release-desktop.yml:189]:
```suggestion
        if: ${{ steps.release-channel.outputs.channel == 'latest' && github.event.inputs.release_type != 'Dry Run' && github.event.inputs.github_release }}
```

**Review by joseph-flinn — CHANGES_REQUESTED** [review]

File: .github/workflows/release-desktop.yml (hunk 1 above)
Comment (michalchecinski) [reply .github/workflows/release-desktop.yml:149]: Oh, you are right. I took a hint from this: https://github.com/bitwarden/clients/blob/master/.github/workflows/release-desktop.yml#L234 But there are no curly braces there.

**Review by joseph-flinn — APPROVED** [review]

**Pull request merged**